suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -223,6 +230,28 @@ export const useAddCoinAccount = () => {
         });
     };
 
+    const navigateToAddCoinDiscoveryRunningScreen = (
+        networkSymbol: NetworkSymbol,
+        flowType: AddCoinFlowType,
+    ) => {
+        clearNetworkWithTypeToBeAdded();
+        navigation.replace(AddCoinAccountStackRoutes.AddCoinDiscoveryRunning, {
+            networkSymbol,
+            flowType,
+        });
+    };
+
+    const navigateToAddCoinDiscoveryFinishedScreen = (
```

Comment:
IMO this hook is getting quite bulky. What about not moving this navigation helper from here to `AddCoinDiscoveryRunningScreen` definition? Personally I would preffer to move also other navigation helpers out of here. This hooks has already more than 400 lines and has responsibility for too many things (I has 11 exports!). But this is also maybe just a personal prefference. I did not do the reviews of PRs of this flow, so maybe there is a good reason for having them here?

Reply:
I did some cleaning there
f471517693